src/lib/packager/clients/pkg_finish.rb

```diff
@@ -288,8 +289,15 @@ def sync_target_sources
     #
     # @see Yast::Packager::CFA::ZyppConf#set_minimalistic!
     def set_minimalistic_libzypp_conf
-      log.info("Setting libzypp configuration as minimalistic")
-      config = Packager::CFA::ZyppConf.new
+      config = if File.file?("/etc/zypp/zypp.conf")
```

Comment:
I think there is a missing chroot prefix, the real path during installation is `/mnt/etc/zypp/zypp.conf`.

So this should be 

```suggestion
      config = if File.file?(File.join(Installation.destdir, "/etc/zypp/zypp.conf"))
```